README.md

```diff
@@ -1,50 +1,34 @@
-* **.eslintrc.json** - contains the course linter configuration
-* **.eslintignore** - contains the course linter ignore configuration
-* **package.json** - contains npm package config
-  * create a `test` script for running tests
-  * create `dbon` and `dboff` scripts for managing the mongo daemon
-* **db/** - contains mongodb files **(should be git ignored)**
-* **index.js** - entry-point of the application
-* **src/** - contains the remaining code
-  * **src/lib/** - contains module definitions
-  * **src/model/** - contains module definitions
-  * **src/route/** - contains module definitions
-  * **src/\_\_test\_\_/** - contains test modules
-  * **main.js** - starts the server
-
-## Feature Tasks  
-For this assignment you will be building a RESTful HTTP server with basic authentication using express.
-
-#### Account
-Create a user `Account` model that keeps track of a username, email, hashed password, and token seed. The model should be able to regenerate tokens using json web token. 
-
-#### Server Endpoints
-* `POST /signup` 
-  * pass data as stringifed JSON in the body of a **POST** request to create a new account
-  * on success respond with a 200 status code and an authentication token
-  * on failure due to a bad request send a 400 status code
-
-## Tests
-* POST should test for 200, 400, and 409 (if any keys are unique)
-
-## Documentation
-In the README.md write documentation for starting your server and making requests to each endpoint it provides. The documentation should describe how the server would respond to valid and invalid requests.
+# 17: Bearer Authentication
+
+This application uses MongoDB to POST and GETnew users/accounts/profiles
+
+## Account Model
+
+Requires a user name, email, and password for each new user instance
+
+## Profile Model
+
+Takes in a first and last name, a bio, avatar, favorite color, location, and a required connection to an account schema.
+
+## Account Routes
+
+`authRouter.post('/signup', jsonParser, (request, response, next))`: Creates a new user account and posts it to our database. Once created, the password is replaced with a hashed password and a token is created for the user. Possible responses to this include:
+
+- 200 for a successful post.
+- 400 if a bad request
+- 409 if any of the existing unique keys are replicated in the post
+
+ID is randomly assigned to each instance by the mongoose middleware.
+
+`authRouter.get('/login', basicAuthMiddleware, (request, response, next))`: Allows a user to receive their profile with the use of a token which was generated upon logging in or signing up. Possible responses include:
+
+- 200 for a successful get.
+- 400 for a bad request.
+
+## Profile Routes
+
+`profileRouter.post('/profiles', bearerAuthMiddleware, jsonParser, (request, response, next))`: Allows a new profile instance to be posted which provides the request.body of the new profile. Possible responses include: 
+
+- 200 for a successful post
+- 400 for an invalid request
+
```

Comment:
Instructions say the following for your README:

```
In the README.md write documention for starting your server and making requests to each endpoint it provides.
```

Tell me how to start and stop my server or run my tests!